src/neo/Network/P2P/RemoteNode.ProtocolHandler.cs

```diff
@@ -294,15 +295,23 @@ private void OnHeadersMessageReceived(HeadersPayload payload)
 
         private void OnInventoryReceived(IInventory inventory)
         {
+            knownHashes.Add(inventory.Hash);
             pendingKnownHashes.Remove(inventory.Hash);
-            if (inventory is Block block)
+            switch (inventory)
             {
-                UpdateLastBlockIndex(block.Index);
-                if (block.Index > NativeContract.Ledger.CurrentIndex(system.StoreView) + InvPayload.MaxHashesCount) return;
+                case Transaction transaction:
+                    system.TxRouter.Forward(new TransactionRouter.Preverify(transaction, true));
```

Comment:
Why `Forward`? I think it should be `Tell`.

Reply:
Have fixed